fedn/fedn/clients/reducer/restservice.py

```diff
@@ -479,7 +480,65 @@ def combiner_stats():
                 except:
                     pass
             return combiner_info
-            return False
+
+        def client_status():
```

Comment:
docstring missing

Reply:
hmm, fixed maybe we need also to check docstring for the whole framework

Reply:
Yep. Prashant has a PR that adds docstrings everywhere.